src/demo-app/a11y/table/table-a11y.html

```diff
@@ -0,0 +1,70 @@
+<div>
+  <section>
+    <h2>Basic Table</h2>
+    <p>Shows name, color and age data.</p>
+    <md-table [dataSource]="basicDataSource">
```

Comment:
I took a quick looks at the guidance for `role="grid"`: https://www.w3.org/TR/wai-aria/roles#grid

Something we missed earlier is that 
> A grid is considered editable unless otherwise specified.

We either need to add `aria-readonly` here, or we need to make the table set it by default. @andrewseguin thoughts?

Reply:
If the table has contents that are editable that would be misrepresentative of the contents. Its really up to the user implementing this to make that decision IMO.

Reply:
I was thinking the table would have something like `editable` that defaults to false, but we can sort that out later.

For now, just add `aria-readonly` to the `md-table` in the demo